src/components/usa-date-picker/usa-date-picker.stories.js

```diff
@@ -0,0 +1,55 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-date-picker.twig";
+import Data from "./usa-date-picker.json";
+
+export default {
+  title: "Components/Date Picker",
+  argTypes: {
+    default_date: {
+      control: "text",
+    },
+    default_value: {
+      control: "text",
+    },
+    min_date: {
+      control: "text",
+    },
+    max_date: {
+      control: "text",
+    },
+    range_date: {
+      control: "text",
+    },
+  },
+};
```

Comment:
This story references default_date in the json but defaultDate in the twig and story exports. Can we remove the values that have underscores?